src/tests/ftest/util/job_manager_utils.py

```diff
@@ -247,6 +300,7 @@ def __init__(self, job, subprocess=False):
         self.ompi_server = FormattedParameter("--ompi-server {}", None)
         self.working_dir = FormattedParameter("-wdir {}", None)
         self.tmpdir_base = FormattedParameter("--mca orte_tmpdir_base {}", None)
+        self.path = path
```

Comment:
This is already defined as `self._path` in `CommandWithParameters.__init__()`.  Please remove.
```suggestion
```